lib/compiler/src/beam_type.erl

```diff
@@ -592,7 +592,11 @@ is_math_bif(log10, 1) -> true;
 is_math_bif(sqrt, 1) -> true;
 is_math_bif(atan2, 2) -> true;
 is_math_bif(pow, 2) -> true;
+is_math_bif(ceilf, 1) -> true;
+is_math_bif(floorf, 1) -> true;
 is_math_bif(pi, 0) -> true;
+is_math_bif(ceil, 1) -> true;
+is_math_bif(floor, 1) -> true;
```

Comment:
Have you checked how `is_math_bif/1` is used?

      case is_math_bif(Math, Ar) of
        true -> tdb_update([{{x,0},float}], Ts);
        false -> tdb_kill_xregs(Ts)
      end;